interface_frontend.py

```diff
@@ -2,7 +2,6 @@
 from pathlib import Path
 from functools import lru_cache
 
-
```

Comment:
Toto je zrovna priklad zmeny, ktera by se s pouzitim blacku vubec nedelala. Mam u sebe kontrolu stylu, ktera me nuti delat dve mezery po importech, jinak mi to obarvuje cervene (a zrovna pred chvili PR, ktery to srovnaval, byl zaclenen). Pokud to odstraneni radku zaclenime, za par tydnu opet poslu PR, ktery sem ten chybejici radek vlozi :(.